Sources/PackageSets/Model/Package.swift

```diff
@@ -0,0 +1,136 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2020 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import struct Foundation.URL
+
+import PackageModel
+import SourceControl
+
+extension PackageSetsModel {
+    /// Package metadata
+    public struct Package {
+        /// Package reference
+        public let reference: PackageReference
+
+        /// Package's repository address
+        public let repository: RepositorySpecifier
+
+        /// Package description
+        public let description: String?
+
+        /// Published versions of the package
+        public let versions: [Version]
+
+        /// The latest published version of the package
+        public let latestVersion: Version?
```

Comment:
Is this the latest stable version, or does it include pre-release semvers?

Reply:
my vote is latest stable, unless non-found and then latest pre-release. examples

given
```
3.0.0-beta.3
3.0.0-beta.2
3.0.0-beta.1
2.1.1
2.1.0
2.0.0
2.0.0-beta.3
2.0.0-beta.2
2.0.0-beta.1
1.0.5
1.0.4
1.0.0
```

latest: 2.1.1

but given

```
1.0.0-beta.3
1.0.0-beta.2
1.0.0-beta.1
```

latest: 1.0.0-beta.3


WDYT? too complicated / confusing / inconsistent?

Reply:
Sure. Let's add a code comment to document the desired behavior?

Reply:
That makes sense.  Yes, a comment and a policy was mainly what I was looking for.

Reply:
done